corehq/apps/data_analytics/models.py

```diff
@@ -79,10 +79,9 @@ class GIRRow(models.Model):
 
     class Meta:
         unique_together = ('month', 'domain_name')
+        index_together = ['domain_name', 'month']
```

Comment:
Curious why this was added. It doesn't appear to be used in the [new GIRRow query](https://github.com/dimagi/commcare-hq/pull/12260/files#diff-4f395d39e2423b9108367531d2d6c672R816), although it does appear that it may have been used by the query that was removed in this file (below).